converters/google/convert_test.go

```diff
@@ -26,22 +27,49 @@ import (
 	"github.com/pkg/errors"
 	"github.com/stretchr/testify/assert"
 	"go.uber.org/zap"
+	"go.uber.org/zap/zapcore"
 )
 
 const testProject = "test-project"
 
-func newTestConverter(convertUnchanged bool) (*Converter, error) {
+type bufferWriteSyncer struct {
+	*bytes.Buffer
+}
+
+func (bws bufferWriteSyncer) Sync() error {
+	return nil
+}
+
+// This returns a logger to allow deterministic testing of the output
+// by omitting the timestamp and calling function.
```

Comment:
❤️

Reply:
This code is pretty similar to the zap.NewExample code - tbh I'm a little unsure it is actually deterministic 😂